app/main.py

```diff
@@ -1,3 +1,43 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: object) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: object) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, number: int) -> "Distance":
+        return Distance(self.km * number)
+
+    def __truediv__(self, other: int) -> "Distance":
+        return Distance(round((self.km / other), 2))
+
+    def __eq__(self, other: object) -> bool:
+        return self.km == other
+
+    def __lt__(self, other: object) -> bool:
+        return self.km < other
+
+    def __le__(self, other: object) -> bool:
+        return self.km <= other
+
+    def __gt__(self, other: object) -> bool:
+        return self.km > other
+
+    def __ge__(self, other: object) -> bool:
+        return self.km >= other
```

Comment:
You can decrease count of these methods using [total ordering](https://docs.python.org/3/library/functools.html#functools.total_ordering)